### ISSUE TITLE
ISVA(Internal) -022 Account does not lock after 4 failed login attempts

### ISSUE THREAD

**Issue by niharaabeywardhana** [niharaabeywardhana/CIAM-ISVA-internal-SSO--Ncinga#21]
### Description
#

Account does not lock after 4 failed login attempts.

###  Step to Reproduce
#

1.Attempt to log in with incorrect credentials(Same User ID/invalid password)
2.Observe the error message displayed after each attempt.

### Expected Result 
#
After the fourth invalid login attempt, the user should receive an account lockout message, and the account should be locked for 15 minutes.

### Actual Result (10/10/2024)
#
After the sixth invalid login attempt, the user should receive an account lockout message, and the account should be locked for 15 minutes.

https://github.com/user-attachments/assets/8a016785-fa93-4987-b672-c48c1c9cf466

**Comment by supunncinga** [comment]
Account related policies has to be maintained in the Active Directory level. nothing to do wit ISVA.

**Comment by niharaabeywardhana** [comment]
According to the BRD,
![image](https://github.com/user-attachments/assets/29edd553-7f1f-4390-ad6e-a2d9b37dfae7)

**Comment by KemaliMunasinghe** [comment]
@supunncinga In the meeting with the bank  with Lasantha, he clearly mentioned that even if certain scenarios cannot be tested with AD, the test scenarios should cover them and if anything that can be covered from our side should be tested as much as possible. Since this is something we can test from our side and can be identifiable as a bug, we think this is a valid bug,

**Comment by supunncinga** [comment]
@KemaliMunasinghe well understood. We can keep this as a bug but the resolution should come from the bank team as they are the one who configure such policies in the Active Directory.

**Comment by KemaliMunasinghe** [comment]
@supunncinga Agreed. @pasanwee99 @niharaabeywardhana Going forward, when you log these kinds of bugs, let's use an appropriate label for the bugs related to their side, so that even if we log bugs, the bugs do not fall under NCINGA bug bucket.
